src/openrct2/interface/console.c

```diff
@@ -727,6 +727,12 @@ static sint32 cc_get(const utf8 **argv, sint32 argc)
         if (strcmp(argv[0], "park_rating") == 0) {
             console_printf("park_rating %d", gParkRating);
         }
+        if (strcmp(argv[0], "park_value") == 0) {
```

Comment:
These need to be `else if`. (In your current implementation, typing `get park_value` yields not only the park value, but also an "Invalid variable" error because it keeps falling through.)

Reply:
Thanks, fixed.